packages/web/src/components/list/MultiDataList.js

```diff
@@ -435,13 +436,20 @@ class MultiDataList extends Component {
 				{this.hasCustomRenderer ? (
 					this.getComponent()
 				) : (
-					<UL className={getClassName(this.props.innerClass, 'list') || null}>
+					<UL
+						className={getClassName(this.props.innerClass, 'list') || null}
+						role="listbox"
+						aria-label={`${this.props.componentId}-items`}
+					>
 						{selectAllLabel ? (
 							<li
 								key={selectAllLabel}
 								className={`${
 									this.state.currentValue[selectAllLabel] ? 'active' : ''
 								}`}
+								role="option"
+								aria-checked={!!this.state.currentValue[selectAllLabel]}
+								aria-selected={!!this.state.currentValue[selectAllLabel]}
```

Comment:
We should avoid code duplication, Can you keep a variable to store `!!this.state.currentValue[selectAllLabel]`, and use it further.